ansible/roles/mongo/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
 - name: Install pymongo 3.5.  This should be changed to install the latest once that is possible.
   pip:
     name: pymongo
-    version: '3.5'
+    version: '3.6'
```

Comment:
If we are changing this, we can remove the `HACK ALERT` comment above and also update the name of this task to simply: `Install pymongo 3.6`

Reply:
Even better - Can we just install the latest pymongo via `pip` and change the name to `Install pymongo`?

Reply:
@mcdonnnj - Did you test installing the latest version of pymongo to see if it works?  That would be ideal, as @jsf9k suggested.

Reply:
@dav3r Thanks for pointing that out, fixed in a2b2af0. @jsf9k Unfortunately since there is already a version of `pymongo` installed when this step runs, it appears that if we do not specify an explicit version it will not change. I tested to verify, and without a version it fails in the same way that prompted this change.

Reply:
Got it- thanks for verifying that and explaining it!